### PR TITLE
fix doc comments

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: lib/testing.dart [hunk 1]
```diff
@@ -22,4 +22,8 @@
 ///           200,
 ///           headers: {'content-type': 'application/json'});
 ///     });
+library http.testing;
```
Comment (jakemac53) [inline lib/testing.dart:25]: Why do we need this?

File: lib/src/base_request.dart [hunk 2]
```diff
@@ -6,6 +6,8 @@ import 'dart:collection';
 
 import 'package:meta/meta.dart';
 
+import '../http.dart';
```
Comment (natebosch) [inline lib/src/base_request.dart:9]: can we do something more narrow?

File: lib/testing.dart (hunk 1 above)
Comment (kevmoo) [reply lib/testing.dart:25]: Something to hang the comment off. Minimizes the chance dartdoc gets confused and thinks one is annotating the import directive

File: lib/src/base_request.dart (hunk 2 above)
Comment (kevmoo) [reply lib/src/base_request.dart]: A bit. Sadly, `get` is in the root lib. Added a `show` so it's clear.

**Review by jakemac53 — APPROVED** [review]

**Pull request merged**